specification/batch/data-plane/readme.md

```diff
@@ -28,7 +28,18 @@ These are the global settings for the Batch API.
 # common 
 title: Batch
 description: Batch Client
-api-version: 2017-01-01.4.0
+api-version: 2017-05-01.5.0

```

Comment:
Windows and it's desperate attempt to get all CRLF when LF will do?

Lemme see if I can eliminate them.